src/Compilers/CSharp/Portable/Binder/Binder_Conversions.cs

```diff
@@ -75,7 +75,7 @@ internal partial class Binder
                         sourceTuple.Type,
                         sourceTuple.Arguments,
                         sourceTuple.Type, // same type to keep original element names 
-                        sourceTuple.HasErrors);
+                        sourceTuple.HasErrors).WithSuppression(sourceTuple.IsSuppressed);
```

Comment:
>.WithSuppression(sourceTuple.IsSuppressed) [](start = 46, length = 42)

Thanks.
We have an issue tracking the implementation of suppressions on tuple literals, so there may be other such bugs.
I assume you're adding a corresponding test in this PR since you fixed this.

Reply:
It is triggered by existing tests after the other relevant changes.

---
In reply to: [260049437](https://github.com/dotnet/roslyn/pull/33648#discussion_r260049437) [](ancestors = 260049437)